include/seqan3/argument_parser/validators.hpp

```diff
@@ -200,11 +238,10 @@ class value_list_validator
      * \throws parser_invalid_argument
      */
     template <std::ranges::forward_range range_type>
-    //!\cond
-        requires std::convertible_to<value_type_t<range_type>, value_type const &>
-    //!\endcond
     void operator()(range_type const & range) const
     {
+        static_assert(std::convertible_to<value_type_t<range_type>, option_value_type>,
+                      "The value type of the given range must be convertible to the validator::value_type.");
```

Comment:
Why did you remove it from the requires clause?

Reply:
because we agreed to use requires only for overload resolution and static_asserts for readable checks

Reply:
I think you still want this in the overload resolution. If option_value_type would be std::vector<int> and I give it a std::vector<int> it takes this overload but then fails with the static assert because int is not convertible to std::vector<int>. But instead you wanted the overload above. I guess this is not an usual use case but not an invalid one.

Reply:
This is  not resolved yet